mlocal/frags/Makefile.stub

```diff
@@ -73,17 +73,18 @@ testall: check test
 .PHONY: rpm
 rpm: dist
 	@echo " BUILD RPM"
-	$(V)(set -x; cd $(SOURCEDIR) && \
+	$(V)(short_version="`echo "$(SHORT_VERSION)" | tr - .`"; \
+	  set -x; cd $(SOURCEDIR) && \
 	  if [ -n "$(RPMPREFIX)" ]; then \
 	    rpmbuild $(RPMCLEAN) -ta \
 	      --define '_prefix $(RPMPREFIX)' \
 	      --define '_sysconfdir $(RPMPREFIX)/etc' \
 	      --define '_localstatedir $(RPMPREFIX)/var' \
 	      --define '_mandir $(RPMPREFIX)/share/man' \
-	      $(SOURCEDIR)/singularity-$(VERSION).tar.gz; \
+	      $(SOURCEDIR)/singularity-$$short_version.tar.gz; \
 	  else \
 	    rpmbuild $(RPMCLEAN) -ta \
-	      $(SOURCEDIR)/singularity-$(VERSION).tar.gz; \
+	      $(SOURCEDIR)/singularity-$$short_version.tar.gz; \
```

Comment:
`$$short_version` ?  Should that not just be `$short_version`?

Reply:
No, in make two dollars translates into one dollar passed to the shell.  It works as it is.

Reply:
Ahh... right. That's to be evaluated in the shell command there.